list-module-callers/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "build": "ncc build src/index.ts --license LICENSE",
+    "test": "vitest"
+  },
+  "dependencies": {
+    "@actions/core": "1.10.1"
+  },
+  "devDependencies": {
+    "@types/node": "20.11.8",
+    "@vercel/ncc": "0.38.1",
+    "typescript": "5.3.3",
+    "vitest": "^1.2.2"
```

Comment:
We use jest, not vitest.
I'm not familiar with vitest.
So we should unify the library to jest if there is no special reason.

Reply:
Replaced with jest https://github.com/suzuki-shunsuke/tfaction/pull/1528/commits/8b1f1cae79e58436244e4dd290be66b1749b76ca

Vitest is a very fast testing framework and its API/config is highly compatible with Jest.
https://vitest.dev/

Reply:
I see. Thank you for your explanation.

I think it's okay to replace Jest with Vitest if Vitest is better than Jest.
But we should not do this in this pull request.
We should open another pull request.